presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -497,7 +498,26 @@ PrestoServer::getExprSetListener() {
 
 std::vector<std::unique_ptr<proxygen::RequestHandlerFactory>>
 PrestoServer::getHttpServerFilters() {
-  return {};
+  std::vector<std::unique_ptr<proxygen::RequestHandlerFactory>> filters;
+
+  if (SystemConfig::instance()->enableHttpAccessLog()) {
+    filters.push_back(
+        std::make_unique<http::filters::AccessLogFilterFactory>());
+  }
+
+  if (SystemConfig::instance()->enableHttpStatsFilter()) {
+    auto filter = getHttpStatsFilter();
```

Comment:
```
httpStatsFilter only works for meta internal? It is not OSS available?
```

Reply:
It's internal for now. I will look in proxygen's implementation and add a similar one for oss.